external/stdk/Makefile

```diff
@@ -131,7 +123,19 @@ $(COBJS): %$(OBJEXT): %.c
 	$(call ARCHIVE, $(BIN), $(OBJS))
 	$(Q) touch .built
 
-.depend: Makefile $(SRCS)
+#################################################
+# certificate
+#################################################
+context:
```

Comment:
Makefile does not execute the context target in the external folder. If you need this, you should add the folder path in Directories.mk.
Could you check an example as shown below?
```
ifeq ($(CONFIG_ENABLE_IOTIVITY),y)
CONTEXTDIRS += $(EXTDIR)$(DELIM)iotivity
endif
```

Reply:
I agree that this solution is more graceful, thanks for suggestion. updated.